html/index.html

```diff
@@ -25,19 +25,17 @@ <h1> <img src="./imgs/logo.png"></h1>
   <img id="banner" src="./imgs/banner.jpg" />
   
   <div class="principal">
-    <h2 class="titulos-centralizados">Sobre a Barbearia Alura </h2>
-    <p>Localizada no coração da cidade a <strong> Barbearia Alura</strong>
-      traz para o mercado o que há de melhor para o seu cabelo e barba.
-      Fundada em 2019, a Barbearia Alura já é destaque na cidade e conquista novos clientes a cada dia.
-    </p>
-    <p id="missao"><em> Nossa missão é:<strong> "Proporcionar auto-estima e qualidade de vida aos
-       clientes".</strong></em> 
-    </p>
-
-    <p>Oferecemos profissionais experientes e antenados às mudanças no mundo da moda.
-      O atendimento possui padrão de excelência e agilidade, garantindo qualidade e satisfação dos nossos
-      clientes. 
-    </p>
+    <div><h2 class="titulos-centralizados"> Sobre a Barbearia Alura </h2>
+      <p>Localizada no coração da cidade a Barbearia Alura traz para o mercado o que há de melhor para o seu cabelo e barba.
+        Fundada em 2019, a Barbearia Alura já é destaque na cidade e conquista novos clientes a cada dia.
+       </p>
+      <p><b> Nossa missão é:</b>"Proporcionar auto-estima e qualidade de vida aos
+         clientes".
+      </p>
+      <p>Oferecemos profissionais experientes e antenados às mudanças no mundo da moda.
+        O atendimento possui padrão de excelência e agilidade, garantindo qualidade e satisfação dos nossos
+        clientes. 
+      </p></div>
```

Comment:
pra ficar mais semântico seu código, coloca o fechamento da div alinhada ao inicio dela
assim:
```suggestion
      </p>
   </div>
```